websecurityscanner/CHANGELOG.md

```diff
@@ -4,6 +4,35 @@
 
 [1]: https://pypi.org/project/google-cloud-websecurityscanner/#history
 
+## 0.2.0
+
+05-15-2019 15:20 PDT
+
+
+### Implementation Changes
+- Retry DEADLINE_EXCEEDED (via synth).  ([#7947](https://github.com/googleapis/google-cloud-python/pull/7947))
+- Remove retries for DEADLINE_EXCEEDED (via synth). ([#7895](https://github.com/googleapis/google-cloud-python/pull/7895))
+- Add routing header to method metadata (via synth).  ([#7605](https://github.com/googleapis/google-cloud-python/pull/7605))
+- Remove classifier for Python 3.4 for end-of-life. ([#7535](https://github.com/googleapis/google-cloud-python/pull/7535))
+- Protoc-generated serialization update. ([#7101](https://github.com/googleapis/google-cloud-python/pull/7101))
+- GAPIC generation fixes. ([#7059](https://github.com/googleapis/google-cloud-python/pull/7059))
+- Pick up order-of-enum fix from GAPIC generator. ([#6882](https://github.com/googleapis/google-cloud-python/pull/6882))
+
+### New Features
+- Generate v1beta. ([#7992](https://github.com/googleapis/google-cloud-python/pull/7992))
+
+### Documentation
+- Updated client library documentation URLs. ([#7307](https://github.com/googleapis/google-cloud-python/pull/7307))
+- Update copyright headers
+
+### Internal / Testing Changes
+- Update noxfile (via synth). ([#7803](https://github.com/googleapis/google-cloud-python/pull/7803))
+- Add session `docs` to nox (via synth). ([#7788](https://github.com/googleapis/google-cloud-python/pull/7788))
```

Comment:
That isn't really the right description for #7788.  Somthing like "Fix 'docs' build under nox." is more accurate.